glue/qt/widget_properties.py

```diff
@@ -237,9 +247,15 @@ def connect_current_combo(client, prop, widget):
     """
 
     def _push_combo(value):
-        idx = widget.findData(value)
-        if idx == -1:
-            return
+        # NOTE: we can't use findData here because if the data is not a 
+        # string, PySide will crash
```

Comment:
I've made an issue to remind me to do this:

https://github.com/glue-viz/glue/issues/774

It will require some more in-depth testing hence why I'm postponing it for now.